### DISCUSSION TITLE
Strip Image Metadata

### DISCUSSION THREAD

**Discussion by darkname** [dlemstra/magick-wasm#47] [Ideas]
Hi, I think there is no way yet to use strip to remove Exif Metadata from Images. Can this be added ? Thanks a bunch
Normally I would use with ImageMagick:
`convert <input file> -strip <output file>`

**Comment by dlemstra** [comment]
I am working on adding more methods/properties to the MagickImage class. All features are already available (as WASM) but they need to the javascript api.

The `strip()` method has just been added to `MagickImage` class and this will become available in the next release.

**Reply by darkname** [reply]
Awesome, thanks a lot

**Reply by dlemstra** [reply]
I will try to publish a new release next week that includes this change.
